url/urlencoded-parser.any.js

```diff
@@ -67,5 +60,4 @@
       }
     })
   }, "response.formData() with input: " + val.input)
-})
-</script>
+});
```

Comment:
This needs a newline.

Reply:
Done, though not recognized by GitHub

Reply:
FWIW, I think it is. If there's no newline there's a red indicator at the end of the line.